scripts/masker/dma-collection-masker

```diff
@@ -264,24 +411,27 @@ def _package_collection(
     Returns:
         Tuple[Path, Path]: The new key file and archive file.
     """
-    file_stem = collection.stem if collection.stem.endswith("zip") else collection.stem.rstrip(".tar")
+    masked_collection_file = _mask_file_collection_key(collection)
+    identity_map[collection.name] = masked_collection_file.name
 
-    logger.info("Zipping %s", collection.stem)
-    archive_file = Path(output_path / f"{file_stem}.zip")
-    key_file = Path(collection.parent / f"{file_stem}.key")
+    logger.info("Zipping %s", masked_collection_file.stem)
+    archive_file = Path(output_path / f"{masked_collection_file.stem}.zip")
+    key_file = Path(collection.parent / f"{masked_collection_file.stem}.key")
     matched_files = (
         list(extract_path.glob("*.csv")) + list(extract_path.glob("*.log")) + list(extract_path.glob("*.txt"))
     )
     with zf.ZipFile(archive_file, "w") as f:
```

Comment:
This is an unintended bug.  I can't think of why we wouldn't supply a value here.  Let's change this in a subsequent PR so that we can isolate this fix from these changes.